metadata.json

```diff
@@ -40,7 +40,7 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 3.0.0 < 5.0.0"
+      "version_requirement": ">= 4.10.10 < 5.0.0"
```

Comment:
With the backward compatibility, this should actually be < 6.0.0 rather than 5.

Reply:
Unfortunately, 3 tests did not pass. I'll try to fix them.

Reply:
Fixed in 96952d4aff023ed04fced7b9effc5b8e141d6fe4